types/iobroker/index.d.ts

```diff
@@ -138,6 +140,17 @@ declare global {
 
             /** Custom settings for this state */
             custom?: Record<string, any>;
+
+            /**
+             * Settings for IOT adapters and how the state should be named in e.g. Alexa.
+             * The string "ignore" is a special case, causing the state to be ignored.
+             */
+            smartName?: string | ({ [lang in Languages]?: string; } & {
```

Comment:
Would it make sense to have an explicit `smartName?: 'ignore' | string | ...` (or is this not allowed?)

Reply:
`'ignore' | string` is identical to `string` (there is an open issue in the TS repo for auto completion purposes).